nupic/utils.py

```diff
@@ -85,16 +85,15 @@ def getSlidingWindow(self):
     return self.slidingWindow
 
 
-  def __setstate__(self, state):
```

Comment:
utils.MovingAverage uses CAPNP already, we have just forgotten to remove this statement.